lib/mox.ex

```diff
@@ -322,6 +322,62 @@ defmodule Mox do
     mock
   end
 
+  @doc """
+  Given a mock and a module implementing the mocked behaviour, stubs all
+  functions described by the behaviour using the implementations in the module.
+
+  This is useful if you want to effectively un-mock a mock or override only a
+  subset of a behaviour.
+
+  ## Examples
+
+      defmodule Calculator do
+        @callback add(integer(), integer()) :: integer()
+        @callback mult(integer(), integer()) :: integer()
+      end
+
+      defmodule CalculatorImplementation do
```

Comment:
This example doesn't work because you are missing `@behaviour`. I am assuming this will be a very common mistake to do so we should probably drop the `@behaviour` requirement or raise if there is none. Thoughts?

Reply:
Ahh, sorry about that. I went back and forth on requiring the `@behaviour` attribute. I think I'd lean towards warning if it's missing. It undermines the "explicit contract" pattern we're trying to support if we don't actually declare the behaviours we're depending on. So as a developer, I think I'd actually want the feedback that I forgot to add the `@behaviour` attribute. It also help us "do the right thing" in `stub_with` in cases where a mock is based on multiple behaviors and is getting stubbed with multiple modules.

Reply:
(and by "warn" I mean `raise`)